ui/app/components/dropdowns/network-dropdown.js

```diff
@@ -291,17 +292,17 @@ NetworkDropdown.prototype.renderCommonRpc = function (rpcList, provider) {
           },
         },
         [
-          rpcTarget === rpc ? h('i.fa.fa-check') : h('.network-check__transparent', '✓'),
+          currentRpcTarget ? h('i.fa.fa-check') : h('.network-check__transparent', '✓'),
           h('i.fa.fa-question-circle.fa-med.menu-icon-circle'),
           h('span.network-name-item', {
             style: {
-              color: rpcTarget === rpc ? '#ffffff' : '#9b9b9b',
+              color: currentRpcTarget ? '#ffffff' : '#9b9b9b',
             },
           }, rpc),
         ]
       )
     }
-  })
```

Comment:
This is a minor issue, but I think it will be helpful to others trying to understand the code: could you make the reversal of the list really explicit by removing the reverse here and doing
```
  const reversedRpcList = rpcList.slice(0).reverse()

  return reversedRpcList.map((rpc) => {
```
at top?

We should be better ordering these on the backend in my opinion. But that is definitely beyond the scope of this task. For now, it would at least be good to make it clear to the reader of the code what is happening.